.github/CODEOWNERS

```diff
@@ -59,4 +59,8 @@ ui/pages/confirmations               @MetaMask/confirmations-ux @MetaMask/confir
 ui/pages/institutional               @MetaMask/mmi
 ui/components/institutional          @MetaMask/mmi
 ui/ducks/institutional               @MetaMask/mmi
-ui/selectors/institutional           @MetaMask/mmi
\ No newline at end of file
+ui/selectors/institutional           @MetaMask/mmi
+
+# Design System to own code for the component-library folder
+# Slack handle: @metamask-design-system-team | Slack channel: #metamask-design-system
+ui/components/component-library      @MetaMask/design-system-engineers
```

Comment:
Seeing an error here for `Unknown owner` but I'm pretty sure its correct https://github.com/orgs/MetaMask/teams/design-system-engineers

![Uploading Screenshot 2024-04-30 at 10.58.23 AM.png…]()

Reply:
Looks like it needs to be created

Reply:
It may not be added as a collaborator for this repo (the team itself) I will check with an admin.

Reply:
It has I think this is an issue with the linter? See handle here @MetaMask/design-system-engineers team

Reply:
This has been resolved by @danjm